docs/content_management/content_organization/copy_move_hide_content.md

```diff
@@ -70,7 +70,18 @@ and select and date and time when the Content item will be hidden:
 
 ![Schedule hiding panel](img/schedule_hiding.png)
 
-A hidden Content item is unavailable for anyone on the front page. It is also grayed out in the Content Tree.
+A hidden Content item is not shown in the frontend when using the default templates. It is also grayed out in the Content Tree.
```

Comment:
If it is not shown with the default templates, what is the special case when it can still be shown? Not retrieved with API but shown?